File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.swift
class Solution {
    func topKFrequent(_ nums: [Int], _ k: Int) -> [Int] {
        var dict = [Int: Int]()
        for num in nums {
            dict[num, default: 0] += 1
        }

        return dict.sorted {
            ($0.value, $0.key) > ($1.value, $1.key)
        }.prefix(k).map { $0.key } 

    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        slow, fast = 0, 1
        n = len(nums) - 1
        while fast <= n:
            if nums[slow] == nums[fast]:
                fast += 1
            else:
                slow += 1
                nums[slow] = nums[fast]
                fast += 1
        return slow + 1
        

